extensions/blocks/podcast-player/podcast-player.php

```diff
@@ -107,8 +107,8 @@ function render_player( $player_data, $attributes ) {
 
 	// Generate object to be used as props for PodcastPlayer.
 	$player_props = array_merge(
-		// Make all attributes available.
-		$attributes,
+		// Add all attributes.
```

Comment:
Should we explain _why_ we need to nest attributes? What was the motivation?

Reply:
I tried to explain it in the PR description. Basically I'd like to put the block attributes into its own field in order to avoid the mix them with the block properties. From my point fo view, and after working on this, it feels cleaner and simpler to follow and understand,